feature/platform/transceiver/tests/zr_input_output_power_test/zr_input_output_power_test.go

```diff
@@ -0,0 +1,416 @@
+package zr_input_output_power_test
```

Comment:
The package line goes below the copyright notice on the top

Reply:
Addressed in the latest commit